servers/xr_server.h

```diff
@@ -71,6 +69,9 @@ class XRServer : public Object {
 		TRACKER_CONTROLLER = 0x02, /* tracks a controller */
 		TRACKER_BASESTATION = 0x04, /* tracks location of a base station */
 		TRACKER_ANCHOR = 0x08, /* tracks an anchor point, used in AR to track a real live location */
+		TRACKER_HAND = 0x10, /* tracks a hand */
```

Comment:
Something we should contemplate is making this enum deprecated. The type is now fully encoded in the different subclasses.

Reply:
I think the enum could still make sense as a way to find the type of trackers you're looking for? Also, we don't have a class that represents `TRACKER_BASESTATION`, although, I don't know how useful that tracker type is anyway.

Reply:
As David pointed out, for now I think we've got to keep it for the `Dictionary XRServer::get_trackers(int p_tracker_types)` method; and for positional trackers that could be unknown/base-station/anchor.